CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ All notable changes to this project will be documented in this file.
 
 Preferably use **Added**, **Changed**, **Removed** and **Fixed** topics in each release or unreleased log for a better organization.
 
+## [2.0.0](https://github.com/quintoandar/validations-engine/releases/tag/2.0.0)
+### Changed
+GchatHelper and message classes were added as the main messenger service. So in this release, the errors can't be sent to Slack. In the next release, we'll add the option to choose between Slack and GChat services.
```

Comment:
Nice, Klebin! 

Maybe we should name not as a new releaser but as a minor patch (2.1.0) to follow the semantic versioning standard. 

Since 2.0.0 make changes that turn 1.0 incompatible and adding the feature to choose between services would keep compatible with 2.0.0 because would be like not choosing at all. Make sense?

Reply:
Totally, Heri! I'll adjust the text!